src/css/support.js

```diff
@@ -46,61 +46,57 @@ define([
 		documentElement.removeChild( container );
 	}
 
-	// Support: node.js jsdom
-	// Don't assume that getComputedStyle is a property of the global object
```

Comment:
> As long as getComputedStyle is available, I'm fine with this.

Great. I'd like to have as many people test it as possible since I'm still worried a little so I'd like to get it into the first beta.